### PR TITLE
Ex-20.

### PR DESCRIPTION
**Pull request by prdj**

### CODE REVIEW

File: 20-PB-program-multiplechoice/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        console.log('Tomatos are blue.');
+        break;
+    case 'green':
+        console.log('Tomatos are green.');
+        break;
+    case 'yellow':
+        console.log('Tomatos are yellow.');
+        break;
+    default:
+        console.log('blue berries')
```
Comment (martl) [inline 20-PB-program-multiplechoice/index.js:19]: don't forget semicolon. this should rather be an error case

File: 20-PB-program-multiplechoice/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        console.log('So bad.');
+        break;
+    case (grade > 2 && grade <= 4 ? grade : null):
+        console.log('Not so bad.');
+        break;
+    case (grade > 4 && grade <= 6 ? grade : null):
+        console.log('Super good.');
+        break;
+    default:
+        console.log('Invalid number.')
```
Comment (martl) [inline 20-PB-program-multiplechoice/index.js:36]: don't forget semicolon.

File: 20-PB-program-multiplechoice/index.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        break;
+    default:
+        console.log('blue berries')
+}
+
+console.log('2.')
+let grade = 7;
+
+switch (grade) {
+    case (grade <= 2 ? grade : null):
```
Comment (martl) [inline 20-PB-program-multiplechoice/index.js:26]: interesting approach :D i never expected that.

File: 20-PB-program-multiplechoice/index.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        console.log('Tomatos are yellow.');
+        break;
+    default:
+        console.log('blue berries')
+}
+
+console.log('2.')
+let grade = 7;
+
+switch (grade) {
```
Comment (martl) [inline 20-PB-program-multiplechoice/index.js:25]: you can just say switch(true)

File: 20-PB-program-multiplechoice/index.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        break;
+        case (percenTage >= 81 && percenTage <= 99 ? percenTage : null):
+        console.log('This is the last push!');
+        break;
+        case (percenTage === 100 ? percenTage : null):
+        console.log('You\'re there. Well done!');
+        break;
+    
+    default:
+        console.log('Invalid number.')
```
Comment (martl) [inline 20-PB-program-multiplechoice/index.js:83]: don't forget semicolon

File: 20-PB-program-multiplechoice/index.js (hunk 3 above)
Comment (prdj) [reply 20-PB-program-multiplechoice/index.js:26]: Yeah I found it online but its not still that clear for me what is going on here...

File: 20-PB-program-multiplechoice/index.js (hunk 2 above)
Comment (prdj) [reply 20-PB-program-multiplechoice/index.js:36]: yeap Thanks

File: 20-PB-program-multiplechoice/index.js (hunk 1 above)
Comment (prdj) [reply 20-PB-program-multiplechoice/index.js:19]: Yeap

**Pull request closed without merging**